packages/ckbcomp/PKGBUILD

```diff
@@ -0,0 +1,37 @@
+# AUR
+# Maintainer: Nissar Chababy <funilrys at outlook dot com>
+# Ex-Maintainer: 	Jeroen Bollen <jbinero at gmail dot comau>
+# Mantainer: Mattia Procopio <matto.astro@gmail.com>
```

Comment:
:joy: it should be your name here @sc74

Reply:
Thanks, but you're the one who should appear, since you're holding the packages. I'm just a little help